setup/pkgs/gfxhat_install.sh

```diff
@@ -16,9 +16,9 @@
 # along with pi-stomp.  If not, see <https://www.gnu.org/licenses/>.
 
 # GFXHat LCD
-pushd $(mktemp -d)
```

Comment:
Shouldn't need to comment these out if the script is never called by setup.sh.  Will still need this for v1

Reply:
I have uncommented the lines in the script.